cli/tests/e2e/snapshots/test_exclude_include/test_exclude_include/exclude-and-include/err.out

```diff
@@ -16,4 +16,4 @@ Some files were skipped or only partially analyzed.
   Scan skipped: 2 files not matching --include patterns, 1 files matching --exclude patterns
   For a full list of skipped files, run semgrep with the --verbose flag.
 
-Ran 4 rules on 1 file: 1 finding.
+Ran 1 rule on 1 file: 1 finding.
```

Comment:
It looks like you added a printout for if rules were missed, but it's not showing here. Isn't that expected tho?

Reply:
yeah – requires https://github.com/semgrep/semgrep-app/pull/11114

Reply:
Why does it require a semgrep-app PR? I would assume that even if you're using local rules (like in these tests) and a rule isn't used, it would print out as so.

Reply:
Oh nevermind, I see this is just about pro rules?